src/flamenco/runtime/program/fd_config_program.c

```diff
@@ -192,6 +226,11 @@ _process_config_instr( fd_exec_instr_ctx_t ctx ) {
 
 int
 fd_config_program_execute( fd_exec_instr_ctx_t ctx ) {
+  do {
+    int err = fd_exec_consume_cus( ctx.txn_ctx, 450UL );
```

Comment:
nit: magic number

Reply:
Oh just define a constant with a link to Solana: https://github.com/solana-labs/solana/blob/2afde1b028ed4593da5b6c735729d8994c4bfac6/programs/config/src/config_processor.rs#L14